src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -3915,6 +3915,11 @@ def _get_uptime_sla(self, enable_validation: bool = False) -> bool:
         """
         # read the original value passed by the command
         uptime_sla = self.raw_param.get("uptime_sla")
+        if uptime_sla:
```

Comment:
could be done with the help of the deprecation mechanism provided by the cli framework

Reply:
Fixed.